packages/gatsby-theme-newrelic/src/components/GlobalStyles/GlobalStyles.js

```diff
@@ -122,6 +122,9 @@ const GlobalStyles = ({ layout }) => (
 
         > li {
           margin: 0.5rem 0;
+          > p {
```

Comment:
You're definitely right, this could affect lists with a margin. I know we have a few lists that have multiple paragraphs in our docs site, so this would effectively remove the margin between them.

The goal here is to try and remove the margin from the paragraph when it is the last child in a container that also contains a margin. This is most affected in markdown where there is much less control over margins on a page.

We do have a specific container for that: https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/MarkdownContainer.js. 

The more I think about it, I think it might be best to make changes there, since markdown is the hardest to control spacing. I think it makes since to copy this rule over from global styles to that `MarkdownContainer` component and specifically target `li > p:last-of-type`.

Would you be willing to make that change? Sorry about being so scatter brained on my part! Its been a hectic couple days for me with the launch of the new site and with GatsbyConf 😄

Reply:
:+1:  no probs

Reply:
@jerelmiller do you have a test/slim project for the theme ? Or how can I test it without having to use the docs project and link the package? (docs build takes ages; then fails)

Reply:
We have been trying our best to figure out how to use `yarn link` to link the theme so we can test on the docs site in real time as well, but no dice either.

We've got a [demo site](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/demo) in this repo we use for trying out changes, which you can use to test. This runs on [localhost:8001](http://localhost:8001]. You can run `yarn start` at the root of this repo to spin it up.

We have a [test markdown document](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/demo/src/pages/build-apps/build-hello-world-app.mdx) you can use to test on if you'd like to see the changes locally. You'll be able to get to that page at http://localhost:8001/build-apps/build-hello-world-app.

Hopefully this helps! Makes me think we need better contributor docs 😄

Reply:
@jerelmiller cool will try that out, as have been doing it blind :eyes: :-D

Reply:
lol was in the code as well!... just did not find it.. an addition to the readme or contributor would definitely be helpful (readme is so long.. you get lost, and contributing is basic stuff)

Reply:
> have been doing it blind

Oh no! Thats no good haha.

> readme is so long.. you get lost, and contributing is basic stuff

This is REALLY HELPFUL feedback! I'll create an issue for us to take a look at this some more so that we can make this easier on future contributors. We really appreciate this!

Reply:
@jerelmiller Maybe a docs sub folder of markdown that the readme links to for config stuff et all (reducing the readme grok)...

Even nicer would then be generating a gatsby site for the docs (or docusaurus / suchlike) :-D

Reply:
> Even nicer would then be generating a gatsby site for the docs

Funny enough, I've had this this thought as well! Our demo site already shows off most of the components/config for the theme. Our [theme documentation](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic) is actually pretty comprehensive, but as we add more and more to the theme, its getting pretty huge. I'd love to repurpose the demo site to be exactly that, a docs site for our theme. Great minds think alike!

I've [filed an issue](https://github.com/newrelic/gatsby-theme-newrelic/issues/304) with your feedback. Feel free to add more to that if you'd like!

Reply:
On another note, the code looks good. I'll check it out locally shortly!

Reply:
@jerelmiller Yeah so using the demo app and changing the mdx this fix (newly force-pushed commit) seems to fix the problem.

Idea could be to add a separate md page for testing basic md styles (instead of changing the demo mdx)

Reply:
> Possibly with a markdown input for directly displaying what your markdown would look like in the actual theme.

Thats a really good idea! We'll look at what we can do to make this process smoother.

Reply:
>  I'd love to repurpose the demo site to be exactly that, a docs site for our theme. 

Dogfood :hotdog: :poodle: ;-)